### PR TITLE
Dashboard flow

### PR DESCRIPTION
**Pull request by BastinJafari**

### CODE REVIEW

File: chapters/dashboard/flow.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+-  Export your flow as YAML or PNG 
+
+
+### Before you start
+
+Before you start, you have to [launch JinaD and connect it to the Dashboard](connect-jinaD.md)
+
+### Composing a flow
+
+![compose flow](flowCompose.gif)
```
Comment (aga11313) [inline chapters/dashboard/flow.md:20]: I think its really good that we have gifs but we might still want some textual description to accompany it

File: chapters/dashboard/flow.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+## Flow Design GUI
+
+##### Our GUI is a visual interface that makes composing flows as easy as browsing Facebook. Design, configure and deploy your flows right from the browser!
```
Comment (aga11313) [inline chapters/dashboard/flow.md]: Should we use a name of another company in here, probably better to stay away?

**Review by aga11313 — COMMENTED** [review]
LGTM, id remove the mention of Facebook (can replace with 'social media' or sth) 

if you have time you can add descriptions to accompany the gifs but i wouldn't prioritize this

File: chapters/dashboard/flow.md (hunk 2 above)
Comment (BastinJafari) [reply chapters/dashboard/flow.md]: Exchanged it with "social media"

File: chapters/dashboard/flow.md (hunk 1 above)
Comment (BastinJafari) [reply chapters/dashboard/flow.md:20]: Added descriptions

**Review by aga11313 — APPROVED** [review]
LGTM

**Pull request merged**